text/0000-minimum-docs-lifecycle.md

```diff
@@ -0,0 +1,62 @@
+# Meta
+[meta]: #meta
+- Name: Define a minimum standard for docs in order to ship the lifecycle
+- Start Date: 4/7/2021
+- Author(s): natalieparellano
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+The project has made improving our documentation a priority for 2021. In order to help deliver on this promise, this RFC proposes defining a minimum standard for documentation that should be met before we're able to ship new versions of the lifecycle.
+
+# Definitions
+[definitions]: #definitions
+
+Docs repo: https://github.com/buildpacks/docs
+
+Migration guides: https://github.com/buildpacks/docs/tree/main/content/docs/reference/spec/migration
+
+Lifecycle-specific content: https://github.com/buildpacks/docs/tree/main/content/docs/concepts/components/lifecycle
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this? Better docs!
+- What use cases does it support? Every persona will be happier about this.
+- What is the expected outcome? Better docs!
+
+# What it is
+[what-it-is]: #what-it-is
+
+Today, our minimum standard for docs gating the implementation team's ability to ship the lifecycle is largely non-existent. We do require that the spec has been released (for which there are release notes), and we also provide release notes for the lifecycle. We've lately been trying to have migration guides at least in PR status to the docs repo. However the migration guides often do not contain enough information to sufficiently explain a feature, because "core" documentation is missing - see [exec.d on Windows](https://buildpacks.io/docs/reference/spec/migration/buildpack-api-0.5-0.6/#execd-on-windows) which explains that Windows is now supported but doesn't link to any further documentation to understand exec.d, because said documentation does not exist.
```

Comment:
nit: this paragraph feels like it should be in "Motivation"

Reply:
Updated in 52e97c49f083bb60861940e1507a3d0fd6fd416f